docs/developers/cosmwasm-contract-deployment.md

```diff
@@ -0,0 +1,61 @@
+# Contract Deployment on CosmWasm with Optimint
+<!-- markdownlint-disable MD013 -->
+
+## Compile the Smart Contract
+
+We will run the following commands to pull down the Nameservice
+smart contract and compile it:
+
+```sh
+git clone https://github.com/InterWasm/cw-contracts
+cd cw-contracts
+git checkout main
+cd contracts/nameservice
+rustup default stable
```

Comment:
`main` is the default branch and `rustup` was configured when setting up dependencies.

```suggestion
cd contracts/nameservice
```